docs/io/output/vpacket_logging.rst

```diff
@@ -0,0 +1,33 @@
+**********************
+Virtual Packet Logging
+**********************
+
+If ``virtual_packet_logging`` is set to ``True`` in either the :ref:`spectrum configuration <spectrum-config>` or as 
+an :ref:`argument to the run_tardis function <optional-input>`, then TARDIS will log information about the
+simulation's :ref:`virtual packets <virtual_packets>`.
+
```

Comment:
Might also wanna add that it can be checked if a simulation has virt packet logging enabled by `sim.virt_packet_logging` property that has boolean value.

Telling this coz this is an important conditional logic that has been used at several places in visualization modules.